app/controllers/expenses_controller.rb

```diff
@@ -2,26 +2,21 @@ class ExpensesController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @authorized_expense_user = Expense.joins(:user).where(["expenses.user_id = ? OR expenses.user_id = ? OR users.invited_by_id = ?",
+    authorized_expense_user = Expense.joins(:user).where(["expenses.user_id = ? OR expenses.user_id = ? OR users.invited_by_id = ?",
                                             current_user.id, current_user.invited_by_id, current_user.id])
-    @search = @authorized_expense_user.order(expense_date: 'DESC').ransack(params[:q])
-    @expenses = @search.result.includes(:user).paginate(page: params[:page], per_page: 15)
+    @search = authorized_expense_user.ransack(params[:q])
+    @expenses = @search.result.order(expense_date: 'DESC').includes(:user).paginate(page: params[:page], per_page: 15)
```

Comment:
Line is too long. [118/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.